lib/order.rb

```diff
@@ -1,18 +1,70 @@
+require 'csv'
+require 'pry'
+
 module Grocery
   class Order
     attr_reader :id, :products
 
+    @@all_orders = []
+
     def initialize(id, products)
       @id = id
       @products = products
+      @tax = 0.075
```

Comment:
I like that you store this in a variable. Since it never changes, maybe a constant would be a good choice here.